BambooTracker/stream/RtAudio/RtAudio.pri

```diff
@@ -5,24 +5,37 @@ HEADERS += \
     $$PWD/RtAudio.h
 
 linux {
-    DEFINES += __LINUX_ALSA__
-    LIBS += -lasound -lpthread
-    DEFINES += __LINUX_PULSE__
-    LIBS += -lpulse-simple -lpulse
-    DEFINES += __UNIX_JACK__
-    LIBS += -ljack
+    defined(__LINUX_PULSE__) {
+        # DEFINES += __LINUX_PULSE__
+        LIBS += -lpthread -lpulse-simple -lpulse
+    }
+    else:defined(__UNIX_JACK__) {
+        # DEFINES += __UNIX_JACK__
+        LIBS += -ljack -lpthread
+    }
+    else {
+        DEFINES += __LINUX_ALSA__
+        LIBS += -lasound -lpthread
+    }
```

Comment:
Sorry for not testing this earlier, but this actually doesn't work. Running
```bash
qmake DEFINES+=__LINUX_PULSE__
```
adds `__LINUX_PULSE__` to `DEFINES`, but falls into the else statement here.

Looking at the [qmake documentation](https://doc.qt.io/qt-5/qmake-test-function-reference.html#defined-name-type), `defined` tests if a *variable called \<x\>* exists, not if a flag has been added to `DEFINES`.

I suggest [`contains`](https://doc.qt.io/qt-5/qmake-test-function-reference.html#contains-variablename-value) instead:
```qmake
contains(DEFINES,__LINUX_PULSE__) {
```
etc.